packages/xo-server-sdn-controller/src/index.js

```diff
@@ -326,15 +326,15 @@ class SDNController extends EventEmitter {
 
   async _objectsUpdated(objects) {
     await Promise.all(
-      map(objects, async (object, id) => {
+      map(objects, object => {
```

Comment:
If `objects` is an array, try to use the native `Array#map`.

Reply:
`objects` is not an array.
I use `lodash#map` for this reason.